src/platform_impl/windows/window.rs

```diff
@@ -874,8 +874,12 @@ impl Window {
   #[inline]
   pub fn request_user_attention(&self, request_type: Option<UserAttentionType>) {
     let window = self.window.clone();
+    let window_flags = self.window_state.lock().window_flags();
+    let is_minimized = window_flags.contains(WindowFlags::MINIMIZED);
```

Comment:
let's move this lock and its check inside the if statement, so locking the mutex can be avoided if the active window is not this window, something like this.

```rs
if window.0 == active_window_handle {
    let window_flags = self.window_state.lock().window_flags();
    let is_minimized = window_flags.contains(WindowFlags::MINIMIZED);
    if !is_minimized {
      return
    }
}
```

Reply:
Thanks for your suggestion, it has been modified